modules/restful_token_auth/src/Plugin/authentication/TokenAuthentication.php

```diff
@@ -89,6 +89,13 @@ protected function extractToken(RequestInterface $request) {
 
     // Access token may be on the request, or in the headers.
     $input = $request->getParsedInput();
+
+    // If we don't have an $key_name on either the URL or the in the headers,
```

Comment:
maybe it's worth also adding in the comment the reason we do this (e.g. Apache's new versions not accepting headers with underscores)